OrbitGl/ImGuiOrbit.cpp

```diff
@@ -589,7 +589,10 @@ void Orbit_ImGui_NewFrame(GlCanvas* a_Canvas) {
   // / h);
 
   // Setup time step
-  io.DeltaTime = a_Canvas->GetDeltaTimeSeconds();
+  // TODO: Does this make sense?
```

Comment:
I'm guessing this makes ImGui assert so I'll merge the change as is.  What is the issue exactly, when does it happen?

Reply:
The assertion that fails is `IM_ASSERT((g.IO.DeltaTime > 0.0f || g.FrameCount == 0)              && "Need a positive DeltaTime!")` in `NewFrameSanityChecks()` in imgui.cpp.
It happens on Windows with debug builds (in my case, building from the Visual Studio solution that cmake generates in bootstrap-orbit.bat).